src/Tickets/Commerce/Order.php

```diff
@@ -298,11 +298,11 @@ public function register_post_type() {
 		/**
 		 * Filter the arguments that craft the order post type.
 		 *
-		 * @see   register_post_type
 		 * @since 5.1.9
 		 *
 		 * @param array $post_type_args Post type arguments, passed to register_post_type()
 		 *
+		 * @see   register_post_type
```

Comment:
Why did we move this?

Reply:
Because Luca's IDE does these weird stuff on its own.